<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionPreferencesForm</class>
 <widget class="QWidget" name="OptionPreferencesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="ignoreBox">
     <property name="text">
      <string>ignore</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Option string:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="stringLine"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Option Argument:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="argumentLine"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
